v1.1/parallel-statement-execution.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Parallel Statement Execution
+summary: The parallel statement execution feature allows parallel execution of multiple independent SQL statements within a transaction.
+toc: false
+---
+
+CockroachDB supports parallel execution of independent INSERT, UPDATE, UPSERT, and DELETE statements within a single transaction. 
+
+To execute statements in parallel, use the `RETURNING NOTHING` clause for the independent SQL statements.
+
+<div id="toc"></div>
+
+## Independent SQL statements
+
+CockroachDB considers SQL statements to be independent if they operate on independent tables and their execution can be safely reordered without affecting their results. 
```

Comment:
I agree. I have been going back-and-forth about how to structure the document. I think the way we usually highlight things is through examples, and I also thought it was important to first clarify what we mean by independent statements so the users have a clear idea of when they can optimize the performance and when they shouldn't. In short, I prioritized the "when" and "why" above the "how". But I agree that in explaining the independent statements, the details about using the RETURNING NOTHING clause are getting buried. Will look into restructuring the document.